docs/terminology.md

```diff
@@ -293,19 +351,21 @@ owed to a [stake](#stake) during redemption is the product of the
 
 ## private key
 
-The private key of a [keypair](#keypair).
+Another word for the [#secret-key]. Solana tools and documentation generally use
+the term 'secret key' instead.
 
-## program
+## program (or on-chain program)
```

Comment:
```suggestion
## program
```

Changing the heading will change the direct anchors to the content. Let's avoid doing so as much as possible